src/main/drivers/barometer/barometer_bmp085.c

```diff
@@ -39,7 +39,7 @@
 
 #ifdef USE_BARO
 
-#if defined(BARO_EOC_GPIO)
+#if defined(BARO_EOC_PIN)
```

Comment:
This won't work for universal targets - there are either no pin defines, or they are defaulted to `NONE`, making these checks ineffective.

Reply:
I understand, however this is a fix to existing broken code (wrong define name used) so that any future refactorings can address this issue.